ambari-server/src/test/java/org/apache/ambari/server/stack/StackManagerMock.java

```diff
@@ -103,10 +111,10 @@ public int hashCode() {
     private Map<String, ServiceModule> cachedCommonServiceModules;
     private Map<String, StackModule> cachedStackModules;
     private Map<String, ExtensionModule> cachedExtensionModules;
-    private Map<String, StackInfo> cachedStackMap;
+    private TreeMap<String, StackInfo> cachedStackMap;
 
     public CachedModules(Map<String, ServiceModule> cachedCommonServiceModules, Map<String, StackModule> cachedStackModules,
-                         Map<String, ExtensionModule> cachedExtensionModules, Map<String, StackInfo> cachedStackMap) {
+                         Map<String, ExtensionModule> cachedExtensionModules, TreeMap<String, StackInfo> cachedStackMap) {
```

Comment:
Can it be declared `NavigableMap` instead of the implementation?  (Same applies to the field and accessor method.)